pythonforandroid/pythonpackage.py

```diff
@@ -202,17 +214,21 @@ def binary_is_usable(python_bin):
     bad_candidates = []
     good_candidates = []
     ever_had_nonvenv_path = False
+    ever_had_path_starting_with_prefix = False
     for p in os.environ.get("PATH", "").split(":"):
         # Skip if not possibly the real system python:
         if not os.path.normpath(p).startswith(
                 os.path.normpath(search_prefix)
                 ):
             continue
 
+        ever_had_path_starting_with_prefix = True
+
         # First folders might be virtualenv/venv we want to avoid:
         if not ever_had_nonvenv_path:
             sep = os.path.sep
-            if ("system32" not in p.lower() and "usr" not in p) or \
+            if ("system32" not in p.lower() and "usr" not in p and
```

Comment:
**Another suggestion:** as before, I think it's more readable this way:
```
            if (
                (
                    "system32" not in p.lower()
                    and "usr" not in p
                    and not p.startswith("/opt/python")
                )
                or {"home", ".tox"}.intersection(set(p.split(sep)))
                or "users" in p.lower()
            ):
```

Reply:
changed it to almost this (I think pep8 wants the binary operators like `or` at the end of the line) I hope you like it, thanks for the suggestion :heart: